src/MainWindow.cc

```diff
@@ -113,9 +117,7 @@ MainWindow::MainWindow(QWidget *parent)
 
         QSettings settings;
 
-        if (!settings.value("user/window/tray", true).toBool()) {
-                trayIcon_->hide();
-        }
+        trayIcon_->setVisible(settings.value("user/window/tray", true).toBool());
```

Comment:
We should use the `userSettings_` object to retrieve user configuration so we don't have to define the settings path every time.